DevelopmentProposals/readme.md

```diff
@@ -0,0 +1,12 @@
+## Overview
+
+This directory contains all the development proposals for OpenSim.
+
+## How to create a proposal
+
+1. Create a new directory under directory ***DevelopmentProposals***.
+2. Copy ***template.md*** into the new directory, and rename it appropriately.
+3. Fill in the proposal details, and add any relevant code snippet files and media files to the directory.
```

Comment:
I am in favor of adding a guideline about this. Also, we could suggest that images be pasted into the PR as a comment rather than as part of the repo.